src/Aspire.Hosting/Oracle/IOracleDatabaseResource.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The file name should match the type name.

Reply:
Ok